packages/typescript-fetch-client2/templates/api.hbs

```diff
@@ -24,10 +23,10 @@ export namespace {{className name}}Api {
 }
 
 /**
- * {{className name}}Api - fetch parameter creator
+ * {{className name}}Api - parameter creator
  * @export
  */
-export const {{className name}}ApiFetchParamCreator = function (configuration?: Configuration) {
+export const ParamCreator = function (configuration?: Configuration) {
```

Comment:
Maybe this shouldn't have an initial capital letter... it isn't a class. Perhaps this is the time to fix that up given that we're renaming it.

Reply:
Note we're also exporting the `ApiFp` function still with its original name. Maybe that should be named `apiFunctions`. Ugh. I don't know.